### DISCUSSION TITLE
Change the library list for my VS Code Session

### DISCUSSION THREAD

**Discussion by gcraig88keys** [codefori/vscode-ibmi#2125] [General]
I tried to set up an action to call a program that will change my connections library list and ran it at the member level, but it didn't change the library list.  We are a software SAAS vendor serving may clients on or systems, so our library list is very basic when we sign on.  We set our libary list based on the client environment we need to sign into.   In RDi, I have various commands configured to call our CL program with a parameter to set my library list the way I needed.    Our library lists are lengthy, so changing it manually would take a long time.     

I want to be able to execute a command like this to set my library list. **CALL CISSETLIBL 'V37PDVFIL'**

**Comment by gcraig88keys** [comment]
I figured it out.   I created a profile and set up a command there. 
![image](https://github.com/codefori/vscode-ibmi/assets/89139309/753431aa-84e2-41d5-910d-19197492b099)

**Comment by chrjorgensen** [comment]
@gcraig88keys Exactly the way to go! 👍
